app/index.js

```diff
@@ -371,6 +407,27 @@ function onClickHangup() {
   stopLocalMedia();
 }
 
+function onClickAVCtrl(isMuteOrPaused, isAudio) {
```

Comment:
@lennart-csio , @karthikbr82 
1. Will we implement AV controls from application like current implementation ? 
2. Or, can our CsioPeerConnection wrapper library should have a function to control AV states ?

Based on the implementation, it seems option 2 will fit with the overall implementation. But, please let me know what you people think ?

Another reason is, I can not send 'applicationLogEvent' from current function scope because 'sendFabricEvent' requires a valid peerconnection object. If we moved it to 'CsioPeerConnection wrapper library' then we can access the peerconnection from that scope.

Reply:
since there might be several PCs (one for each other participant in the call), IMO the CsioPeerConnection class is the wrong place. but we can put it into the main.js of the library, which knows all the PCs and the stream

Reply:
of course that doesnt solve the csio integration problem.. so we need to either expose the list of PCs, so this can be handled completely in application, or fire an event in main.js to trigger the csio integration to send these events

Reply:
You will also have to integrate the sendFabricevent to report the mute/unmute pause/resume https://www.callstats.io/api/#enumeration-of-fabricevent